### ISSUE TITLE
Supress output in python

### ISSUE THREAD

**Issue by chatziko** [gchers/fbleau#10]
There should be an option (even default maybe?) to completely supress all output in the python lib.

**Comment by gchers** [comment]
Good point! Will do after merging #faster-knn branch (which makes everything a little faster):)
